src/SFML/Graphics/Texture.cpp

```diff
@@ -259,55 +260,50 @@ bool Texture::loadFromStream(InputStream& stream, const IntRect& area)
 bool Texture::loadFromImage(const Image& image, const IntRect& area)
 {
     // Retrieve the image size
-    int width  = static_cast<int>(image.getSize().x);
-    int height = static_cast<int>(image.getSize().y);
+    Vector2i size(image.getSize());
+
+    IntRect absArea(area.getAbsoluteRect());
```

Comment:
```suggestion
    Vector2i size = image.getSize();

    IntRect absArea = area.getAbsoluteRect();
```

Reply:
Can't use this style for size, need to convert from Vector2u

Reply:
Can't use this style for size, need to convert from Vector2u